DQMOffline/Configuration/scripts/cmsswSequenceInfo.py

```diff
@@ -0,0 +1,532 @@
+#!/usr/bin/env python3
+import os
+import re
+import time
+import shutil
+import sqlite3
+import tempfile
+import functools
+import subprocess
+from collections import namedtuple
+from collections import defaultdict
+from multiprocessing.pool import ThreadPool
+
+Sequence = namedtuple("Sequence", ["seqname", "step", "era", "scenario", "mc", "data", "fast"])
+
+# We use two global thread pools, to avoid submitting from one Pool into itself.
+tp = ThreadPool()
+stp = ThreadPool()
```

Comment:
@smuzaffar @silviodonato We could make the number of threads configurable here quite easily for the unit tests.